x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/Info.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import lightTheme from '@elastic/eui/dist/eui_theme_light.json';
+
+const ItemRow = styled('div')`
```

Comment:
probably use `styled.div` instead of `styled('div')` for consistency with other components

Reply:
Updated in 147f974ffd.